src/app/expansion-panel/expansion-panel.constants.ts

```diff
@@ -0,0 +1,17 @@
+export type ExpansionPanelConfig = {
+  startsExpanded: boolean;
+  buttonStyles: string;
+  contentContainerStyles: string;
+};
+
+export const defaultConfig: ExpansionPanelConfig = {
+  startsExpanded: true,
+  buttonStyles: 'material-icons-outlined tce-text tce-text-slate-600 hover:tce-bg-slate-200 hover:tce-rounded',
+  contentContainerStyles: '',
+};
+
+export const questionPanelConfig: ExpansionPanelConfig = {
```

Comment:
This should live on the carbon-estimator-form for now, passing it in to the configOverride input when used for help display.

Reply:
I've moved this to the carbon-estimator-form. Let me know if it's not in quite the right place/files.